src/config/store/devtools.prod.ts

```diff
@@ -0,0 +1 @@
+export const StoreDevModules = [];
```

Comment:
Could we add a comment here to say:  "This disabled DevTools in Production mode, overriding the defaults in `devtools.ts`"

I want to be sure we are describing these new configs, as they are not very straightforward on their own.